src/components/organize/events/EventList/EventListItem.tsx

```diff
@@ -24,24 +21,12 @@ const EventListItem = ({ event, hrefBase }: EventListItemProps): JSX.Element =>
                         { title || activity.title }
                     </Typography>
                     <Typography color="textPrimary" variant="body2">
-                        <FormattedDate
-                            day="numeric"
-                            month="long"
-                            value={ startTime }
-                        />
-                        { `  ` }
-                        <FormattedTime
-                            value={ startTime }
+                        <ZetkinDateTime
+                            datetime={ removeOffset(start_time) }
```

Comment:
I remember we talked about doing the `removeOffset()` inside of `ZetkinDateTime`. Is there any reason why we shouldn't?

Reply:
I do not remember that there was a specifik reason, so can move this to ZetkinDateTime insted!